apps/challenges/serializers.py

```diff
@@ -48,13 +65,6 @@ class Meta:
                   'is_public', 'is_active', 'codename')
 
 
-class DatasetSplitSerializer(serializers.ModelSerializer):
```

Comment:
Also there is no need of reordering this serializer here. One of the main reason can be it will actually confuse someone of why this model was involved in this commit. Also in the master, its model `DatasetSplit` is not ordered alphabetically. So in future(may be) we will open an issue where in you can reorder this.

One thought for keeping the serializer as it is it actually tells the history of how the model came into existence. Dont know how much of this is true, but yeah one can simply infer that yeah `Challenge` model was introduced before `ChallengePhase` model.